backend/src/data/migrations/20220811102918_fix_db_associations.ts

```diff
@@ -0,0 +1,87 @@
+import { Knex } from 'knex';
```

Comment:
looks like you've made this branch from `fix/GRHB100-fix-DB-associations`

Reply:
No, I just copied this file, because Icould not test migration properly)

Reply:
Do not worry) There will be no conflicts (I think so :))

Reply:
I'm not worrying at all)
But if you have issues with corrupted migration pull, because of migration from `fix/GRHB100-fix-DB-associations` on your current branch, then there is a better (and I think correct) way to overcome this issue -- we have `migrate:dev:down` script in BE `package.json` file so you've just needed to rollback `20220811102918_fix_db_associations.ts` (by running `npm run migrate:dev:down 20220811102918_fix_db_associations.ts` inside backend folder) from the branch where it was done (actually it's better always to rollback migrations from work branches when you're swapping to another task)

--
Just imagine if you need to make 3-4 tasks with migration in one day - and how it would be if for each time you'd copy-pasting each created migration

Reply:
I tried that, but got same error, that migration corrupted because there is no that migration file from fix.
So, do I need to remove file with previous migration (from fix)?

Reply:
Btw, thanks for an advice

Reply:
Try to check in your db in `migrations` folder what migrations were ran for it and compare them with migrations that should be in `development`, maybe you also skipped another one 🤔

Reply:
Yep, all migrations from development were executed and two new (which I worked with)